pytun_pmd3/wintun.py

```diff
@@ -179,6 +188,10 @@ def raise_last_error() -> None:
     raise Exception(get_error_message(get_last_error()))
 
 
+async def wait_for_event(handle: HANDLE) -> None:
+    await asyncio.get_running_loop().run_in_executor(None, lambda: kernel32.WaitForSingleObject(handle, INFINITE))
```

Comment:
Might be worth to verify the returned value is valid:
https://learn.microsoft.com/en-us/windows/win32/api/synchapi/nf-synchapi-waitforsingleobject

Reply:
done